xla/service/gpu/gpu_compiler.cc

```diff
@@ -1982,11 +1982,12 @@ absl::StatusOr<std::unique_ptr<Executable>> GpuCompiler::RunBackend(
     std::unique_ptr<HloModule> module, se::StreamExecutor* stream_exec,
     const CompileOptions& options) {
   Thunk::BinaryMap dnn_compiled_graphs;
+#if GOOGLE_CUDA  // Note: this feature is not yet available on ROCM
```

Comment:
We should minimize ifdefs in the code.
We could for example try to move this into NvptxCompiler. (Maybe you'll have to add a new virtual method.)

Reply:
yes, I have added a respective virtual function to NvPtxCompiler